thirty-three-across/thirty-three-across-htb.js

```diff
@@ -197,7 +334,7 @@ function ThirtyThreeAcrossHtb(configs) {
         if (ComplianceService.isPrivacyEnabled()) {
             var gdprStatus = ComplianceService.gdpr.getConsent();
 
-            gdprConsent = Object.assign(gdprConsent, {
+            gdprConsent = Utilities.mergeObjects(gdprConsent, {
```

Comment:
Interesting, why the switch to `Utilities.mergeObjects`?

Reply:
for IE9 compatibility. `Utilities.mergeObjects` is their util function, and its supporting IE9.